src/components/Header/HeaderNavItem/index.tsx

```diff
@@ -0,0 +1,22 @@
+import { usePathname } from "next/navigation";
+import { HTMLAttributeAnchorTarget } from "react";
+import styles from "./header-nav-item.module.css";
+
+type Props = {
+  label: string;
+  href: string;
+  target?: HTMLAttributeAnchorTarget;
+};
+
+const NavItem: React.FC<Props> = ({ label, href, target = "_self" }) => {
+  const pathname = usePathname();
+  const isActive = pathname === href;
+
+  return (
+    <a href={href} target={target}>
+      <li className={isActive ? styles.active : styles.default}>{label}</li>
+    </a>
```

Comment:
We can have `<li><a></a></li>` if we wanna follow the best HTML semantics practice. See https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Document_and_website_structure.

Reply:
you are absolutely right, going to fix it.